tutorial/chapter/5/index.md

```diff
@@ -4,20 +4,22 @@ title: "Properties and function calls"
 
 ## Property access
 
-Property access is written just like JavaScript. The only difference is that in
-Squiggle these will throw exceptions if `object[property]` is `undefined`.
+Property access is written just like JavaScript. The difference is that Squiggle will throw an exception unless the Squiggle code `obj has key` is true.
```

Comment:
The `obj has key` code is what you would use to check if it's safe to access the property, but I should call that out to be clear.